Fix roads

gcperk opened the pull request:
This PR shifts the get_roads() function created in create_vectors into an external function to be used for later purposes in sample design. 

@ateucher - not likely to need much attention but adding as review if you want!

ateucher commented on R/get_roads.R:
```diff
@@ -0,0 +1,67 @@
+  }
+
+
+  if (nrow(roads) > 0 & nrow(fsr) > 0) {
```
What if only one of them has `> 0` rows? Do you still want them or should it throw a warning? Could replace with an OR, since `bind_rows()` should still work even if one df has 0 rows as I think it still has the columns?

```suggestion
  if (nrow(roads) > 0 || nrow(fsr) > 0) {
```

ateucher reviewed (approved):
I think this looks good Gen! Will you add a test or two? Also I think it's good practice to delete the commented code before merging :)

gcperk commented:
thanks @ateucher, yes I will certainly add some test (I added a issue to make sure it is captured #30) Nice catch on the or statement, yes I think that is much better as I forgot that option was not currently flagged! I also removed the comments from the get_roads().
